sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/samples/Sample1_ExecutePipeline.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Identity;
+using NUnit.Framework;
+using Azure.Analytics.Synapse.Artifacts;
+using Azure.Analytics.Synapse.Artifacts.Models;
+using Azure.Core.TestFramework;
+
+namespace Azure.Analytics.Synapse.Samples
+{
+    /// <summary>
+    /// This sample demonstrates how to execute piplines in Azure Synapse Analytics using synchronous methods of <see cref="PipelineClient"/>.
+    /// </summary>
+    public partial class ExecutePipelines
+    {
+        private PipelineClient PipelineClient;
+
+        [OneTimeSetUp]
+        public void CreateClient()
+        {
+            // Environment variable with the Synapse workspace endpoint.
+            string workspaceUrl = TestEnvironment.WorkspaceUrl;
+
+            #region Snippet:CreatePipelineClient
+            // Create a new Pipeline client using the default credential from Azure.Identity using environment variables previously set,
+            // including AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, and AZURE_TENANT_ID.
+            PipelineClient client = new PipelineClient(endpoint: new Uri(workspaceUrl), credential: new DefaultAzureCredential());
```

Comment:
Typically we would define the referenced variables inside the snippet so that it can be easily copy/pasted and run. For example:

```c#
// Replace the string below with your actual workspace url.
//@@ string workspaceUrl = "my-workspace-url";
```